Description: Bugfix for fvwm3-menu-desktop
  Removes unneeded output from fvwm3-menu-desktop --get-menus all which
  causes erroneous entries in the graphical configuration utility.
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: https://github.com/fvwmorg/fvwm3/pull/593
Last-Updated: 2021-06-21
--- a/bin/fvwm-menu-desktop.in
+++ b/bin/fvwm-menu-desktop.in
@@ -475,7 +475,6 @@ Standard output is a series Fvwm command
         fh.write(current_theme)
         fh.close()
 
-    if force:
         for ent in desktop_entries:
             style_id = ent.getStartupWMClass()
             if not style_id:
